stoservice/src/test/java/com/tutrit/stoservice/repository/CarRepositoryTest.java

```diff
@@ -0,0 +1,89 @@
+package com.tutrit.stoservice.repository;
+
+import com.tutrit.stoservice.bean.Car;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class CarRepositoryTest {
+
+    CarRepository carRepository;
+    Car car1;
+    Car car2;
+    Car car3;
+
+    @BeforeEach
+    void setUp() {
+        carRepository = new CarRepository();
+        car1 = new Car("id1", "brand1", "model1", "1g", "mod1", "engine1", 2001);
+        car2 = new Car("id2", "brand2", "model2", "2g", "mod2", "engine2", 2002);
+        car3 = new Car("id3", "brand3", "model3", "3g", "mod3", "engine3", 2003);
+        CarRepository.cars.add(car1);
+        CarRepository.cars.add(car2);
+        CarRepository.cars.add(car3);
+    }
+
+    @Test
+    void createCar() {
+        int expected = carRepository.count() + 2;
+        carRepository.createCar(new Car("id4", "brand4", "model4", "4g", "mod4", "engine4", 2004));
+        carRepository.createCar(new Car("id5", "brand5", "model5", "5g", "mod5", "engine5", 2005));
+
+        assertEquals(expected, carRepository.count());
```

Comment:
You need to check not only that the Sheet has changed, but also that your object is not in this sheet.

Reply:
you meant that the volume was added and the sheet changed